src/perf/bin/drvmain.cpp

```diff
@@ -74,16 +74,12 @@ SecNetPerfCtlUninitialize(
         void
     );
 
-void* __cdecl operator new (size_t Size) {
-    return ExAllocatePool2(POOL_FLAG_NON_PAGED, Size, QUIC_POOL_PERF);
-}
-
 _Ret_maybenull_ _Post_writable_byte_size_(_Size)
 void* __cdecl operator new (size_t Size, const std::nothrow_t&) throw(){
     return ExAllocatePool2(POOL_FLAG_NON_PAGED, Size, QUIC_POOL_PERF);
 }
 
-void __cdecl operator delete (_In_opt_ void* Mem) {
+void __cdecl operator delete (/*_In_opt_*/ void* Mem) {
```

Comment:
why are you removing `_In_opt_`?

Reply:
Because `new.h` doesn't include it, and VS 16.10 is complaining that the attributes don't match. I can't change the headers, so the option is to uncomment here or suppress the warning. This is better then suppression IMO.